portal/static/js/main.js

```diff
@@ -1327,8 +1327,15 @@ var fillContent = {
         };
     },
     "websiteConsentScript": function() {
-        var patientId = $("#wcsPatientId").val();
-        var entryMethod = $("#wcsEntryMethod").val();
+        var getUrlParameter = function(name) {
+            name = name.replace(/[\[]/, '\\[').replace(/[\]]/, '\\]');
+            var regex = new RegExp('[\\?&]' + name + '=([^&#]*)');
```

Comment:
Not a huge fan of regular expressions, [if there are alternatives](https://css-tricks.com/snippets/javascript/get-url-variables/) (not sure if this requires a polyfill...).